features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```
@@ -26,6 +26,7 @@
   "session_data.cleanup.clean_logged_out_sessions_at_immediate_cycle": true,
   "session_data.cleanup.enable_pre_session_data_cleanup": true,
   "session_data.cleanup.pre_session_data_cleanup_thread_pool_size": "20",
+  "session_data.storage_optimization.enable_context_optimization": true,
```

As we are going to reuse for all the optimizers 
```suggestion
  "session_data.storage_optimization.enable": true,
```